src/chart/Treemap.tsx

```diff
@@ -108,8 +108,6 @@ const horizontalPosition = (row: any, parentSize: number, parentRect: TreemapNod
     child.width = Math.min(rowHeight ? Math.round(child.area / rowHeight) : 0, parentRect.x + parentRect.width - curX);
     curX += child.width;
   }
-  // what's z
```

Comment:
Amazing comment. While this seems to be a bug fix, I am yet worried whether some other place in recharts magically expects this to be set. Please do manually test the TreeMap in the Demo, and investigate whether this is used anywhere.

Typescript does not help a lot due to the large amount of any's in the code base.

Reply:
Thanks for your review.
Sure, I tested it in the Demo. I checked the result rendered and element attributes.

Reply:
+1 to Niko - I'm going to wait to merge this until I pull it down and play with it (at least just for some more confidence if nothing else). Looks like a non-breaking change from the naked eye (and testing). Thanks